packages/svgalib.rb

```diff
@@ -0,0 +1,55 @@
+require 'package'
+
+class Svgalib < Package
+  description 'SVGAlib is a low-level graphics library for Linux. It augments the C programming language, which doesn\'t provide support for graphics.'
+  homepage 'https://www.svgalib.org/'
+  version 'e698'
+  source_url 'https://github.com/ryanmcgrath/svgalib-1/archive/e698d8c11f938bb6039a1791dc54c620c228d308.tar.gz'
+  source_sha256 'ff39a2b375c36b688934d4be88193b1fd02d3809fa398dbf51d0fcd182087452'
+
+  binary_url ({
+  })
+  binary_sha256 ({
+  })
+
+  def self.patch
+    system "sed -i 's,#ifndef __GNUC__,#ifdef __GNUC__,g' threeDKit/quickmath.h"
+    system "sed -i '/ldconfig/d' src/Makefile threeDKit/Makefile Makefile lrmi-0.9/Makefile.lrmi"
+    system "sed -i -e 's,#!/usr/bin/perl,#!#{CREW_PREFIX}/bin/perl,g' -e 's,/usr/lib/kbd,#{CREW_PREFIX}/lib/kbd,g' utils/svgakeymap"
```

Comment:
The file `/usr/local/lib/kbd` doesn't exist so not sure why you are even worried about it.  With that said, could this be a missing dependency?

Reply:
To be honest, I have no idea. It doesn't seem to use the keymap in any way at all, so I have no idea why it would be needed.